templates/phoenix_elixir/files/.platform.app.yaml

```diff
@@ -0,0 +1,49 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+name: app
+
+# The runtime the application uses.
+type: elixir:1.9
+
+build:
+    flavor: none
```

Comment:
Is there a flavor to override?  If we have to override it on even our first template, it means we should push back on Engineering and remove flavors entirely.  (Which is what we had planned to do in the first place for non-PHP, but...)

Reply:
@Crell this  is what I've found the flavor assumes so far  https://pr-1198-pj4okaq-652soceglkw4u.eu-3.platformsh.site/languages/elixir.html#assumptions

Reply:
It's removed on phoenix because it breaks the compile step, but in the "vanilla" elixir template I'll push in a moment, the flavor works fine.

Reply:
We should still be pushing back on Engineering.  We decided after Node that we didn't want to use flavors anymore as it was too much magic, so I don't know why that's there at all.

Reply:
That's fair, I think Ori's for removing it entirely.

Reply:
So I guess then I'll remove flavor mentions from the simple template too for now.